docs/usermanual.md

```diff
@@ -210,7 +210,7 @@ This mechanism and the bidirectionality plugin cannot be simultaneously activate
 ##### Configuration command topic
 
 ```text
-/{{apikey}}/{{deviceid}}/configuration/commands
+/json/{{apikey}}/{{deviceid}}/configuration/commands
```

Comment:
Is configuration command considered a command? (from the point of view of not-usage of the prefix)

Reply:
IMHO, yes, it's a kind of command